.changeset/chilled-jeans-sing.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/math-input": patch
+---
+
+Minor fix to ensure that the keypadElement is being provided to webapp.
```

Comment:
**thought**: For changelogs in Perseus, I try to write them without referencing the consuming app too much.  Can we adjust this slightly to be agnostic?  Something like "Minor fix to ensure that the keypadElement is always provided to mobile keypad consumers" ?